Write 2012

jinglescode opened the pull request:
lesson 201.2 on multi-signature transaction

Commit:
commit from ci pipeline

Commit:
commit from ci pipeline

nelsonksh committed:
Merge pull request #13 from Andamio-Platform/staging

v1.0.0-alpha

nelsonksh committed:
Merge pull request #14 from Andamio-Platform/main

sync

jinglescode committed:
lesson 2012 draft done

Zagita21 commented:
@jinglescode 
2012.md:
* 15: recommend changing "This is a security feature..." to "This can be used as a security feature..."
* 19: (grammar correction):
	-   when we require inputs from multiple wallets eg Wallet A (50 ADA) and Wallet B (50 ADA) sending to Wallet C.
	(etc.)
* 35: recommend clarifying source, eg by adding "(provided by @MeshJS/React)" after "We will use `CardanoWallet)`
* 53: Is it better to replace "UTXO" with "UTxO" so that it is consistent with library style? (appears throughout)
* In the following sections it is not clear enough exactly where the code snippets are going (inside createTransaction?, or startMinting()?, or...?).  Although it finally says something on line 171, and the final code is included later, it is a bit confusing if the learner is just trying to follow directly from the steps. So I think it needs more clear signposting before (or otherwise at least to tell the learner to check the final code for position).  
* 74: Perhaps we can clarify, eg by starting sentence: "In this case, you can think of..."
* 78: fix: "To create a new..." (add 'To') 
*  ... may need to also clarify that the [...] should be replaced with the actual mnemonic words by the learner
* 96: suggested edit, add "also": "... script will also determine..."
* 105: should we link to where learner can find out more information about metadata? (eg Cardano CIP, and/or Mesh object definition?)
* 128: doesn't read quite right. I think you mean something like "... and the receiver address to which payment should be made".
* 130: fix: "that HAVE" (not has)
* 136: first time "bank address" is used. For consistency it could be used also before, or otherwise replaced with something like "payment address", or clarified some other way
* 173: consistency: "`App Wallet`"

Zagita21 committed:
Update 2012.md

made most (nearly all) edits from previous comment.

Still to do:
- revise for clarity
- add suitable link for 96

Commit:
review 2012 (#2), edits & cleanup 2012

Commit:
review 2012 (#2) fix conflicts

jinglescode committed:
Merge pull request #14 from Zagita21/write-2012

review edits for Write 2012 and fix conflicts
